db/version_set.h

```diff
@@ -331,14 +333,33 @@ class VersionStorageInfo {
   }
 
   // REQUIRES: This version has been saved (see VersionSet::SaveTo)
-  using BlobFiles = std::map<uint64_t, std::shared_ptr<BlobFileMetaData>>;
+  using BlobFiles = std::vector<std::shared_ptr<BlobFileMetaData>>;
   const BlobFiles& GetBlobFiles() const { return blob_files_; }
 
+  // REQUIRES: This version has been saved (see VersionSet::SaveTo)
```

Comment:
Just realize: seems `SaveTo` is a method of `VersionBuilder`?

Reply:
Oops, nice catch :) Will fix this across the board (there are some preexisting occurrences as well)